gridstatus/ercot.py

```diff
@@ -3240,6 +3241,21 @@ def ambiguous_based_on_dstflag(df):
                     self.default_timezone,
                     ambiguous=ambiguous,
                 ) - pd.Timedelta(hours=1)
+            except pytz.AmbiguousTimeError:
+                # Sometimes ERCOT handles DST end by putting 25 hours in HourEnding
+                # which makes IntervalStart where HourEnding >= 3 an hour later than
+                # they should be. We correct this by subtracting an hour.
+                doc.loc[doc["HourEnding"] >= 3, "Interval Start"] = doc.loc[
```

Comment:
to be safe, i think we probably want to also assert or wrap this logic with a conditional that confirms that there is a HourEnding that equals 25 in the data

Reply:
Good idea, added the assertion.